specification/streamanalytics/resource-manager/Microsoft.StreamAnalytics/preview/2017-04-01-preview/subscriptions.json

```diff
@@ -115,6 +166,71 @@
         }
       }
     },
+    "QueryTestingResult": {
+      "description": "The result of the query testing request.",
+      "allOf": [
```

Comment:
You seem to define error twice? One by inheritance here and then directly in the "properties" section. Are both needed?